### PR TITLE
Adding JER/JEC options to Framework

### PR DESCRIPTION
**Pull request by kmei-cms**

### CODE REVIEW

File: Framework/include/BJet.h [hunk 1]
```diff
@@ -143,8 +145,9 @@ class BJet
     }
 
 public:
-    BJet() 
-        : bjets_(nullptr)
+    BJet( std::string myVarSuffix ) 
+        : myVarSuffix_(myVarSuffix)
```
Comment (cmadrid1) [inline Framework/include/BJet.h]: Can you add a default value here for "myVarSuffix".

File: Framework/include/MakeMVAVariables.h [hunk 2]
```diff
@@ -114,7 +115,8 @@ class MakeMVAVariables
     }
 
 public:
-    MakeMVAVariables(const bool verb = false) : verb_(verb)
+    MakeMVAVariables(const bool verb = false) : verb_(verb) :
+        myVarSuffix_(myVarSuffix)
```
Comment (cmadrid1) [inline Framework/include/MakeMVAVariables.h]: Right now you are not passing in a "myVarSuffix". In fact this shouldn't compile.

File: Framework/include/BJet.h (hunk 1 above)
Comment (kmei-cms) [reply Framework/include/BJet.h]: Sounds good will do.

File: Framework/include/MakeMVAVariables.h (hunk 2 above)
Comment (kmei-cms) [reply Framework/include/MakeMVAVariables.h]: Right, this makes sense. As I said, I have no code that actually uses MakeMVAVariables, but yeah I can change that.

**Comment by kmei-cms** [comment]
I think I fixed the issues for BJet.h and MakeMVAVariables.h here in the most recent commit :)

File: Framework/include/MakeMVAVariables.h [hunk 3]
```diff
@@ -114,7 +115,8 @@ class MakeMVAVariables
     }
 
 public:
-    MakeMVAVariables(const bool verb = false) : verb_(verb)
+    MakeMVAVariables(const bool verb = false, std::string myVarSuffix = "") : verb_(verb) :
```
Comment (cmadrid1) [inline Framework/include/MakeMVAVariables.h]: This line has an error. It didn't compile. ": verb_(verb) :"

**Comment by cmadrid1** [comment]
In general @kmei-cms can you make sure your code compiles before you try to merge it into github?

**Pull request merged**